### PR TITLE
Fix chat notification highlight words split regex

### PR DESCRIPTION
**Pull request by Kamielvf**
This fixes an issue where the chat notifications plugin would not split word higlights when input as `some,highlight` even though they are supposed to be valid.

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/chatnotifications/ChatNotificationsPlugin.java [hunk 1]
```diff
@@ -71,6 +71,8 @@
 	@Inject
 	private RuneLiteProperties runeLiteProperties;
 
+	private final Pattern SPLIT_REGEX = Pattern.compile("\\s*,\\s*");
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/chatnotifications/ChatNotificationsPlugin.java]: also this should be static final

Comment (deathbeam) [reply runelite-client/src/main/java/net/runelite/client/plugins/chatnotifications/ChatNotificationsPlugin.java]: oh and also above fields

**Comment by Nightfirecat** [comment]
This should probably add a test to ensure this case is properly working now.

File: runelite-client/src/main/java/net/runelite/client/plugins/chatnotifications/ChatNotificationsPlugin.java (hunk 1 above)
Comment (raiyni) [reply runelite-client/src/main/java/net/runelite/client/plugins/chatnotifications/ChatNotificationsPlugin.java]: `public static final Splitter SPLITTER = Splitter.on(",").trimResults();`

File: runelite-client/src/main/java/net/runelite/client/plugins/chatnotifications/ChatNotificationsPlugin.java [hunk 2]
```diff
@@ -115,7 +117,7 @@ private void updateHighlights()
 
 		if (!config.highlightWordsString().trim().equals(""))
 		{
-			String[] items = config.highlightWordsString().trim().split(", ");
+			String[] items = SPLIT_REGEX.split(config.highlightWordsString().trim());
```
Comment (raiyni) [inline runelite-client/src/main/java/net/runelite/client/plugins/chatnotifications/ChatNotificationsPlugin.java]: `SPLITTER.splitToList(config.highlightWordsString())`

**Comment by deathbeam** [comment]
Any update on this @Kamielvf ?

**Comment by Kamielvf** [comment]
I will get to this later when I have time @deathbeam either today or tomorrow

**Pull request merged**